scripts/gitlab/check_runtime.sh

```diff
@@ -106,6 +109,23 @@ then
 	spec_version or or impl_version have changed in substrate after updating Cargo.lock
 	please make sure versions are bumped in polkadot accordingly
 	EOT
+
+	# Now check if any of the substrate changes have been tagged B2-breaksapi
+	(
+		cd "${SUBSTRATE_CLONE_DIR}"
+		substrate_changes="$(sanitised_git_logs "${SUBSTRATE_PREV_REF}" "${SUBSTRATE_NEW_REF}")"
+		echo "$substrate_changes" | while read -r line; do
+			pr_id=$(echo "$line" | sed -E 's/.*#([0-9]+)\)$/\1/')
+
+			if has_label 'paritytech/substrate' "$pr_id" 'B2-breaksapi'; then
+				echo "Substrate change labelled with breaksai. Labelling"
+				exit
+				#github_label "B2-breaksapi"
```

Comment:
everything looks like in its right place to me. just on how i understand it: a label should be added here and the script should exit with a failure code since the spec/impl versions haven't been bumped. the later exit is correct though and was missing.

Reply:
These two exits (line 122 and 127) were erroneously left by me during testing. Are you saying we *do* want to exit immediately after labelling? I didn't think we would.

Reply:
As far as i understand in this stage of the script, there are no changes in the runtime files (and therefore also not in the versions files) and thus this script should end here.

In fact the script would exit already a bit earlier (exit 0) when there are no changes to the versions files - regardless of the labels there.

So I think it would be sufficient if the script exitted 1 when the first breaksapi label on substrate is detected (while labelling breaksapi as well) - the spec/impl versions should be bumped.

And even if there are no labels on substrate, at that point the versions have been altered in substrate and should be here as well. Therefore also passing back a 1.